src/Bicep.Core/Parsing/ReplParser.cs

```diff
@@ -60,7 +60,7 @@ public override ProgramSyntax Program()
             if (next.Type == TokenType.NewLine)
             {
                 // skip blank lines between entries
-                reader.Read();
+                nodes.Add(reader.Read());
```

Comment:
Up for discussion/ideas on this if needed.
Reason being, for parsing diagnostics, the spans seem to adhere to the original input text, i.e. containing new-lines (rather than the `ProgramSyntax.ToString()` representation, which didn't include newline tokens). This caused some issues with annotations not aligning with the actual input text positions

Reply:
Ignore this! Turns out I didn't need to change this, and can just use the fullContent string length to calculate the offset.